Bio/Align/Applications/_Muscle.py

```diff
@@ -22,9 +22,10 @@ class MuscleCommandline(AbstractCommandline):
     >>> muscle_exe = r"C:\Program Files\Aligments\muscle3.8.31_i86win32.exe"
     >>> in_file = r"C:\My Documents\unaligned.fasta"
     >>> out_file = r"C:\My Documents\aligned.fasta"
-    >>> muscle_cline = MuscleCommandline(muscle_exe, input=in_file, out=out_file)
+    >>> muscle_cline=MuscleCommandline(muscle_exe,input=in_file,out=out_file)
     >>> print(muscle_cline)
-    "C:\Program Files\Aligments\muscle3.8.31_i86win32.exe" -in "C:\My Documents\unaligned.fasta" -out "C:\My Documents\aligned.fasta"
+    "C:\Program Files\Aligments\muscle3.8.31_i86win32.exe"
+    -in "C:\My Documents\unaligned.fasta" -out "C:\My Documents\aligned.fasta"
```

Comment:
It looks like you've inserted a line break. Unfortunately that will break the doctest, so in this situation we just have to accept some occasionally long lines.